openingd/openingd.c

```diff
@@ -671,8 +671,7 @@ static u8 *funder_channel(struct state *state,
 	if (!tx) {
 		/* This should not happen: we should never create channels we
 		 * can't afford the fees for after reserve. */
-		negotiation_failed(state, true,
-				   "Could not meet their fees and reserve");
+		negotiation_failed(state, true, "Could not meet our fees and reserve");
```

Comment:
I'm confused, the comment above the creation of `tx` indicates that this is "their" commitment, which would indicate that they can't meet the agreed upon fees (fees are non-owned) and they can't meet the reserve we impose on them. Or is the comment above also switched around?

Reply:
@cdecker okay, first thanks for your attention here.

Thing is the code behaves unpredictably unstable. Sadly its not my machine alone as we can see at the Travis output. Sometimes I run in Valgrind issues, then I recompile and they disappear without code changes, both times clean build. :(

1. I will remove the non working commit, so you can see the same Valgrind issues on Travis again that I linked in IRC yesterday  1079737 fix: openingd memory leak *state on failures    
(so this PR will again be 'please have a look at my broken test')

2. I did the renaming of the exception, because in my testcase `test_opening_tiny_channel` this was raised by `[l1]` which is the founder of the channel and not `[l2]`. Maybe I misinterpreted the code, but I thought the message relates to the situation that the founders own fees (l1) cannot be met and not the responders (l2).

> /* we should never create channels we can't afford the fees for after reserve.

I left this comment untouched, it tells me that there is something wrong with the founders commitment tx (meaning not **theirs**)